istio/README.md

```diff
@@ -18,6 +18,10 @@ Follow the instructions below to install and configure this check for an Agent r
 
 Istio is included in the Datadog Agent. [Install the Datadog Agent][2] on your Istio servers or in your cluster and point it at Istio.
 
+#### Envoy
+
+If you want to monitor the Envoy proxies in Istio, configure the [Envoy integration][23].
```

Comment:
Just as a clarification, does `configure the Envoy integration` mean a customer should configure both Istio and Envoy integrations? Or is this saying that a customer should configure the Envoy integration instead of the Istio integration?

Reply:
It's saying to configure envoy if you want to monitor envoy. Envoy is the sidecar proxy used in istio.